src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/BoosterDetails/BoosterDetailsViewController.swift

```diff
@@ -0,0 +1,72 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import UIKit
+
+class BoosterDetailsViewController: DynamicTableViewController {
+
+	// MARK: - Init
+
+	init(
+		viewModel: BoosterDetailsViewModel
+	) {
+		self.viewModel = viewModel
+		
+		super.init(nibName: nil, bundle: nil)
+	}
+
+	@available(*, unavailable)
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	// MARK: - Overrides
+
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		
+		setupTableView()
+	}
+	
+	override func viewWillAppear(_ animated: Bool) {
+		super.viewWillAppear(animated)
+
+		if let dismissHandlingNC = navigationController as? DismissHandlingNavigationController {
+			dismissHandlingNC.restoreOriginalNavigationBar()
+		}
+	}
```

Comment:
We need to move the `markBoosterRuleAsSeen()` from `HealthCertifiedPersonViewController` here, as it now should only be marked as read if this screen was shown. I think it would make sense to do it in `viewDidAppear` on this screen.